content/Video_Display.md

```diff
@@ -105,11 +105,11 @@ hidden by the text box.
 
 As we saw in the description of the STAT register, the LCD Controller cycles
 through the different modes in a fixed order. If we set the STAT bits
-in such a way that they would interrupt the LCD Controller at two
+in a way that they would interrupt the LCD Controller at two
 consecutive modes, then the second interrupt will not trigger. So for example,
-if the Mode 0 and LY=LYC interrupts are enabled, the LY=LYC interrupt may
-not trigger. This is because the LY=LYC interrupt triggers during Mode 2
-(which usually goes right after Mode 0).
+if we enable the interrupts for Mode 0 and LY=LYC (which triggers during Mode 2),
+the LY=LYC interrupt will not trigger unless we are lucky and there is a Mode 1
```

Comment:
`unless we are lucky` seems very out of context here. Maybe `unless a Mode 1 happens` but it'll need some rephrasing specifying the once per frame thing.